models.py

```diff
@@ -120,7 +120,8 @@ class Category(BaseMixin, Deactivatable, db.Model):
     help_text = db.Column(db.String())
     criteria = db.relationship('Criterion', backref='category', lazy=True)
 
-    def __init__(self, title=None, help_text=None):
+    def __init__(self, id=None, title=None, help_text=None):
+        self.id = id
```

Comment:
Updated some of these initializer methods to take id because we will be using the id from the CMS as a source of truth/unique identifier for these resources. e.g. If the CMS says a category has id 5, then it will have id 5 in the backend here as well.